include/drivers/pwm.h

```diff
@@ -267,7 +270,9 @@ static inline int z_impl_pwm_pin_enable_capture(const struct device *dev,
  * @param pwm PWM pin.
  *
  * @retval 0 If successful.
- * @retval Negative errno code if failure.
+ * @retval -EINVAL if invalid function parameters were given
+ * @retval -ENOTSUP if PWM capture is not supported
+ * @retval -EIO if IO error occurred enabling PWM capture
```

Comment:
This should refer to disabling, not enabling.
And it looks like a word like "during" or "while" is missing after "occurred" in this description (similarly in the one for `pwm_pin_enable_capture()`).

Reply:
@anangl Fixed, thanks.